### ISSUE TITLE
Cannot produce jar from Gradle due to media dependencies

### ISSUE THREAD

**Issue by BrianErikson** [EmergentOrganization/cell-rpg#84]
Due to the way JARs are structured, Gradle puts all of the `../android/assets` folder contents into the root classpath of the JAR. This causes two issues:
- property.settings cannot be referenced properly in multiple deployment states
  - need to use `Gdx.files`, not `Java.io`
- Cannot use [`FileHandle.list()`](https://libgdx.badlogicgames.com/nightlies/docs/api/com/badlogic/gdx/files/FileHandle.html#list--) to get contents of a directory due to the files being zipped into a JAR. This is in reference to dynamically loading all sounds located in the `sounds/30s_arcade_loops/` directory for the `BgSoundController` class.
  - One proper way to fix this would be to build an index in a `txt` file that we can reference during runtime to get a list of files located in a target asset directory

**Comment by 7yl4r** [comment]
This sounds kind of like the issue we had in TheOregonTraj, where we had to wrap file-references in a function getFileDir(). Do we need to set up a class for that in this project?

**Comment by BrianErikson** [comment]
So, the end result is that I had to copy the entire assets dir to the parent directory of the JAR, and then reference it through LIBGDX normally. Since gradle exports everything inside of the `assets` directory to the classpath of the JAR, I had to put everything inside of a subdirectory called `resources`. 

When launching the JAR, the assets directory will be extracted to the **working directory**. This means that if your working directory parameter in IntelliJ is blank, it will attempt to copy the files to the root of the project _I think_. Below is an example of a working JAR configuration.
![screenshot from 2015-10-21 20 51 25](https://cloud.githubusercontent.com/assets/7645647/10646802/87a249b6-7835-11e5-8b97-8afcc289c35d.png)
